devito/ir/support/basic.py

```diff
@@ -750,14 +761,14 @@ def __init__(self, exprs):
             # reads
             for j in retrieve_terminals(e.rhs):
                 v = self.reads.setdefault(j.function, [])
-                mode = 'R' if not q_inc(e) else 'RI'
+                mode = 'RI' if e.is_Increment and j.function is e.lhs.function else 'R'
```

Comment:
why did change to `not is_inq` to `is_inq`?

Reply:
oh, I also swapped 'R' and 'RI' 

If I remember correctly, just to stay within the 90chars limit